src/System.Reflection.DispatchProxy/src/System/Reflection/DispatchProxyGenerator.cs

```diff
@@ -363,6 +363,11 @@ internal void AddInterfaceImpl(Type iface)
 
                 foreach (MethodInfo mi in iface.GetRuntimeMethods())
                 {
+                    // Skip regular/non-virtual instance methods, static methods, and methods that cannot be overriden
+                    // (methods that cannot be overriden includes default implementation of other interface methods).
```

Comment:
nit: "methods ... includes" => "methods ... include"? Or did I read it incorrectly?